plugins/hls-explicit-imports-plugin/hls-explicit-imports-plugin.cabal

```diff
@@ -32,3 +32,16 @@ library
   default-extensions:
     DataKinds
     TypeOperators
+
+test-suite tests
```

Comment:
Wow the plugin had no test suite??? Many thanks for adding it, a great contribution in its own

Reply:
It would be great to add it to the github workflow to run them in ci, adding an entry like

https://github.com/haskell/haskell-language-server/blob/a0cd84b4f6c6e48eb56e135faf5f9b1e3139886c/.github/workflows/test.yml#L207-L209

Reply:
OK. I'll fix.